### ISSUE TITLE
modifies-parameter flags on _ (ignored) parameters

### ISSUE THREAD

**Issue by akerl-unpriv** [mgechev/revive#355]
**Describe the bug**
The modifies-parameter rule flags "modification" of the `_` variable, despite the fact that this indicates ignoring that parameter. This causes issues in functions which ignore an input parameter and then drop a return value from a call.

**To Reproduce**

revive.toml:

```
severity = "warning"
confidence = 0.5
[rule.modifies-parameter]
```

main.go:

```
package main

func doubleFunc() (string, error) {
    return "what", nil
}

func testFunc(_ string) {
    var err error
    _, err = doubleFunc()
    if err != nil {
        panic(err)
    }
}

func main() {
    testFunc("foo")
}
```

Run: 
```shell
revive -config revive.toml main.go
```

**Expected behavior**
No warning is raised, because the `_` variable is ignored.

**Logs**

**Desktop (please complete the following information):**
 - OS: Mac
 - Golang: 1.12

**Comment by chavacava** [comment]
Hi @akerl-unpriv, thanks for reporting this problem. I'll fix it shortly.

**Comment by chavacava** [comment]
@akerl-unpriv could you check if the PR #356 solves the problem? Thanks.
